external_libraries/TLSF-2.4.6/COPYING

```diff
@@ -17,7 +17,7 @@ USA.
 As a special exception, including TLSF header files in a file, or linking with
 other files objects to produce an executable application, is merely considered
 normal use of the library, and does *not* fall under the heading of "derived
-work". Therfore does not by itself cause the resulting executable application
+work". Therefore does not by itself cause the resulting executable application
```

Comment:
A legit fix, but external_libraries are imported from elsewhere. I'm not totally sure of the procedure but I think this fix should be proposed upstream, in addition to (or maybe instead of) fixing it here.

Same a couple lines down for "GNU [General] Public License."

Reply:
yes, let's not modify 3rd-party code unless absolutely necessary